pom.xml

```diff
@@ -1152,15 +1155,24 @@
           <name>code-coverage</name>
         </property>
       </activation>
+      <dependencies>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+            <scope>test</scope>
+        </dependency>
+      </dependencies>
       <build>
         <pluginManagement>
           <plugins>
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
               <configuration>
-                <!-- Append surefire.argLine property populated by JaCoCo plugin -->
-                <argLine>-Xmx1024m -Dfile.encoding=UTF-8 @{surefire.argLine}</argLine>
```

Comment:
This property is used only by the prepare-agent goal. As this goal is replaced with instrument and restore-instrumented-classes goals, it is not needed anymore. This makes the java agent a little bit less flexible, but it is price that needs to be payed for the offline instrumentation. In the unlikely case you want to do some custom configurations to the java agent, there are two other ways how to do this (see http://www.eclemma.org/jacoco/trunk/doc/offline.html).